addons/safemode/stringtable.xml

```diff
@@ -24,7 +24,7 @@
             <Polish>Zwolnij bezpiecznik</Polish>
             <Czech>Uvolnit pojistku</Czech>
             <Hungarian>Biztonsági kapcsoló eltolása</Hungarian>
-            <Russian>Снять с предохранителя</Russian>
+            <Russian>Снято с предохранителя</Russian>
```

Comment:
this should be in `TookOffSafety` string, not `TakeOffSafety` https://github.com/acemod/ACE3/blob/9154a92af505e63cf7fcbf0c82ece05b9ef1dec1/addons/safemode/stringtable.xml#L59